xmind/tests/test_workbook_element.py

```diff
@@ -0,0 +1,751 @@
+import logging
+from . import base
+from unittest.mock import patch, MagicMock, call, Mock
+from xmind.core.workbook import WorkbookElement
+from xmind.core.const import TAG_WORKBOOK, NAMESPACE, XMAP, NS_FO, NS_XHTML, NS_XLINK, NS_SVG, TAG_SHEET, ATTR_VERSION
+
+
+class TestWorkbookElement(base.Base):
+
+    def setUp(self):
+        super(TestWorkbookElement, self).setUp()
+
+        self._workbook_mixin_element_init = self._init_patch_with_name(
+            '_mixin_init', 'xmind.core.topic.WorkbookMixinElement.__init__')
+        self._sheet_element_init = self._init_patch_with_name(
+            '_sheet_element_init', 'xmind.core.sheet.SheetElement.__init__')
+
+        self._setAttributeNS = self._init_patch_with_name(
+            '_setAttributeNS', 'xmind.core.Element.setAttributeNS')
+
+        self._getChildNodesByTagName = self._init_patch_with_name(
```

Comment:
From methods you're mocking here it looks like you give a chance to go further in ```getSheets``` method during ```__init__```
I would recommend you to mock ```getSheets``` and return some value here to block ```__init__``` to go further

> ```#``` Initialize WorkbookElement need contains at least one SheetElement
>        if not self.getSheets():
>            sheet = self.createSheet()
>            self.addSheet(sheet)